packages/boxel/addon/components/boxel/thread/index.hbs

```diff
@@ -4,6 +4,7 @@
   </div>
 
   <div class="boxel-thread__content-wrapper">
+    {{!-- Note that tabindex=0 is applied to ensure that this scroll context can be reached and operated by keyboard --}}
```

Comment:
I suppose a more pleasant looking way to do this is similar to how a thread on discord is navigated. There, each message is highlighted when using a keyboard, instead of the whole scrollable area. This makes is possible to focus on the elements inside of the thread message accessible without having to tab through the entire thread.

Reply:
So if there's time & priority for improvements, that's what I would recommend. Otherwise, your PR looks good to me.

Reply:
I think we can't do this right now because the messages in here are not meant to be interacted with and adding more tabbable items just makes it harder for non-pointer users. For the most part messages are just static text.In the future where there are richer interactions similar to discord (reply to message etc) this might be good.